src/nile/core/account.py

```diff
@@ -65,7 +78,10 @@ def deploy(self):
 
     def send(self, to, method, calldata, max_fee, nonce=None):
         """Execute a tx going through an Account contract."""
-        target_address, _ = next(deployments.load(to, self.network)) or to
+        target_address, _ = next(deployments.load(hex_address(to), self.network)) or to
```

Comment:
This was not compliant with the pattern from my side because all the methods should work with int. The solution I used is just to remove the hex_address here and add the normalize_number to the address as input in deployments.load. This allows passing to as an alias (I didn't notice this, docstring for params would be super helpful as you suggested).

Reply:
Excellent! Yeah, I just wanted to throw out a workaround solution to potentially aid in the adjustment. Works well now!